set.mm

```diff
@@ -467236,6 +467245,280 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
+      UEUWRUUHAUWSUUTUUMWGXFUWTUUTUUMWJYSEYIDJHUUKQYIVDTRVSVTABHYKEIGYIDJYRUXKU
+      ROFNPYNAQUTUBUEYIVDRTYKVDUDWMVTUXFUXQUXNYRYKUXFAUWRUUTUXRAUWSUUTUUMXEUXFU
+      UEUWRUUHAUWSUUTUUMWGXFUWTUUTUUMWJABHYKEIGYIDJYSUUKUROFNPYNAQUTUBUEYIVDRTY
+      KVDUDWMVTWRUXFUYDUYHGUXFYOUUEUWRUUTUYDUYHVBUXFAYOAUWSUUTUUMXEUEVIUXFUUEUW
+      RUUHAUWSUUTUUMWGWHUXFUUEUWRUUHAUWSUUTUUMWGXFUWTUUTUUMWJYRYSEYLYIDJHUUKQYI
+      VDTRYLVDYBWTXAYCXBUXFUXQUXCYRYKUXFUULUUGYSYKUXEUUMWOWRWRYDAUUEUUHUWOUWRUW
+      PUWJUWQUWOAUWJUUHAUWMUWJUBJBGXJVIVPUWPUUGBUWKAUUHWOAUWNUUHAUWMUWNUBJBGXPV
+      IVPXKUWJUWQUWOUIJUUGGXLXMXNXOXQAYRBVFZWEZUKVOZGUMZYRVBZYMYRYKWFZYRVBUKJUY
+      KUYLJVFZWEZUYNWEZYMUYMYKWFZYMUYLEWFZGUMZUYOYRUYRAYMDVFZUYPUYSVUAVBAUYJUYP
+      UYNXEAVUBUYJUYPUYNAYPVUBAYOYPUEYIHYIVDVHVIDYIYMRYMVDYEVIXRUYKUYPUYNWJABHY
+      KEIGYIDJYMUYLUROFNPYNAQUTUBUEYIVDRTYKVDUDWMVTUYRUYMYRYMYKUYQUYNWOWRUYRVUA
+      UYMYRUYRUYTUYLGUYRYOUYPUYTUYLVBUYRAYOAUYJUYPUYNXEUEVIUYKUYPUYNWJEYMYIJHUY
+      LQYIVDTYMVDYFXNXAUYQUYNWOYDXBUYKUWJYRUWKVFZUYNUKJXDZAUWJUYJAUWMUWJUBJBGXJ
+      VIVPUYKYRBUWKAUYJWOAUWNUYJAUWMUWNUBJBGXPVIVPXKUWJVUCVUDUKJYRGXLXMXNXQYG
+      $.
+  $}
+
+  ${
+    $d G a b k p q x $.  $d M a b k p q x $.  $d N a b k p q $.
+    $d V a b k p q x $.  $d a b k p q ph x $.
+    quslmod.n $e |- N = ( M /s ( M ~QG G ) ) $.
+    quslmod.v $e |- V = ( Base ` M ) $.
+    quslmod.1 $e |- ( ph -> M e. LMod ) $.
+    quslmod.2 $e |- ( ph -> G e. ( LSubSp ` M ) ) $.
+    $( If ` G ` is a submodule in ` M ` , then ` N = M / G ` is a left module,
+       called the quotient module of ` M ` by ` G ` .  (Contributed by Thierry
+       Arnoux, 18-May-2023.) $)
+    quslmod $p |- ( ph -> N e. LMod ) $=
+      ( vx vq vp cqg co cfv cv a1i eqid wcel wa vk va cqs cplusg csca cbs cvsca
+      vb cec cmpt c0g cvv clmod cqus wceq ovexd qusval quslem csubg wer lsssubg
+      clss syl2anc eqger syl fvexi cgrp lmodgrp adantr simprl simprr grpcl cnsg
+      syl3anc wi cabl lmodabl ablnsg 3syl eleqtrrd eqgcpbl ercpbl simpr1 simpr2
+      wbr w3a simpr3 qusvscpbl imaslmod ) AECBMNZUCCUDOZCUEOUFOZCUGOZUAJEJPWJUI
+      UJZCDECUKOZKLUBUHAJWJCDWNEULUMDCWJUNNUOZAFQECUFOUOZAGQWNRACBMUPHUQGWLRWKR
+      WMRWORAJWJCDWNEULUMWPAFQWQAGQWNRACBMUPHURAJUBPZUHPZLPZKPZWKWJWNELKABCUSOZ
+      SZEWJUTACUMSZBCVBOZSZXCHIXEBCXERVAVCWJCEBGWJRVDVEEULSAECUFGVFQWNRAWTESZXA
+      ESZTZTCVGSZXGXHWTXAWKNZESAXJXIAXDXJHCVHVEVIAXGXHVJAXGXHVKEWKCWTXAGWKRVLVN
+      ABCVMOZSWRWTWJWEWSXAWJWETWRWSWKNXKWJWEVOABXBXLAXDXFXCHIXEBCXERVAVCAXDCVPS
+      XLXBUOHCVQCVRVSVTWRWSWTXAWKWJCEBGWJRWKRWAVEWBAUAPZWLSZWRESZWSESZWFZTJEWJW
+      LDUGOZWMWRWNBXMCDWSGWJRWLRWMRAXDXQHVIAXFXQIVIAXNXOXPWCFXRRWNRAXNXOXPWDAXN
+      XOXPWGWHHWI $.
+
+    ${
+      $d V k u v x y z $.  $d F k u v y z $.  $d G k u v x y z $.
+      $d M k u v x y z $.  $d N k u v x y z $.  $d ph k u v x y z $.
+      quslmhm.f $e |- F = ( x e. V |-> [ x ] ( M ~QG G ) ) $.
+      $( If ` G ` is a normal subspace in ` M ` , then the "natural map" from
```

Comment:
What is a "normal subspace"? There's a similar notion from topology, but it's not relevant.
Did you mean a "submodule" (that is required in order to have a factor-module correctly defined)?

Also the last sentence probably should be "... is a left module homomorphism from **M** to M / G ."

Reply:
You're right, thanks! I've fixed that in the last commit.